roles/lib_openshift/library/oc_obj.py

```diff
@@ -1531,14 +1533,11 @@ def run_ansible(params, check_mode=False):
         if state == 'list':
             return {'changed': False, 'results': api_rval, 'state': 'list'}
 
-        if not params['name']:
```

Comment:
Is there a way we can have this fail if not 'name' or 'selector' ? If we don't have a name we should have a selector, otherwise we wouldn't know what to delete.